Classes/Runtime/Action/EmailAction.php

```diff
@@ -119,7 +124,7 @@ public function perform(): ?ActionResponse
             );
             return $response;
         } else {
-            $this->mailerService->getMailer()->send($mail);
+            $this->getMailerService()->getMailer()->send($mail);
```

Comment:
This all could even be simplified to:
```suggestion
            $mailerService = $this->objectManager->get(MailerService::class)
            if ($mailerService === null) {
                throw new ActionException('The "neos/symfonymailer" doesn\'t seem to be installed, but is required for the EmailAction to work!', 1503392532);
            }
```

because `ObjectManagerInterface::get()` has a proper return type definition.
And this would only throw if `$testMode` is not active.

But in practice it doesn't really matter I suppose!

Reply:
Hmm, the object manager throws an exception:
``` 
Object "Neos\SymfonyMailer\Service\MailerService" is not registered.
```

Reply:
I'd keep it as it is. We could also catch the exception, but IMHO this makes very clear at the top of the method, that you need the package here.

Reply:
Yes, I agree! Thanks
+